subsys/bluetooth/host/conn_internal.h

```diff
@@ -29,9 +29,9 @@ enum {
 	BT_CONN_BR_PAIRING_INITIATOR,	/* local host starts authentication */
 	BT_CONN_CLEANUP,                /* Disconnected, pending cleanup */
 	BT_CONN_AUTO_PHY_UPDATE,        /* Auto-update PHY */
-	BT_CONN_SLAVE_PARAM_UPDATE,	/* If slave param update timer fired */
-	BT_CONN_SLAVE_PARAM_SET,	/* If slave param were set from app */
-	BT_CONN_SLAVE_PARAM_L2CAP,	/* If should force L2CAP for CPUP */
+	BT_CONN_PERIPHERAL_PARAM_UPDATE,/* If periph param update timer fired */
```

Comment:
```suggestion
	BT_CONN_PERIPHERAL_PARAM_UPDATE,/* If peripheral param update timer fired */
```

Reply:
They are already using abbreviations like "param", hence why I left it abbreviated. They are not Doxygen comments so they will not appear in the main doc.